docs/UserGuide.md

```diff
@@ -296,19 +296,24 @@ Refer to [properties of contact](#properties-of-contact) on the accepted formats
 
 ###  Update contact's photo: `updatephoto`
 
-Updates the photo of an existing contact.
+Besides the ability to edit a contact's information, CampusConnect also provides you with the freedom to customise how it looks in your address book.
 
-Format: `updatephoto INDEX path/NEW_PHOTO_PATH`
+This feature involves the command: `updatephoto`, which updates the photo of an existing contact.
 
-Examples:
-* `updatephoto 1 path/C:/photos/new_johndoe.jpg`
-  * Updates the photo for the 1st person with a new image from the specified path.
-* `updatephoto 2 path/D:/images/new_betsycrowe.png`
-  * Updates the photo for the 2nd person with a new image from the specified path.
+Format: `updatephoto PERSON_INDEX [path/NEW_PHOTO_PATH]`
+
+<box type="info" icon=":fa-solid-magnifying-glass:">
+
+Below are some examples on how to use the command:
+* `updatephoto 1 path/C:/photos/new_johndoe.jpg`: Updates the photo for the first person in the address book with a new image from the specified path.
+* `updatephoto 2 path/D:/images/new_betsycrowe.png`: Updates the photo for the second person in the address book with a new image from the specified path.
+
+</box>  
 
 <box type="warning">
 
-You are required to input the absolute path to the photo for this command. If you use an invalid path, a default photo will be chosen for your specified contact.
+* You should provide a positive integer that is smaller than or equal to the number of contacts currently displayed in CampusConnect for `PERSON_INDEX`. 
+* You are required to input the absolute path to the photo for this command. If you use an invalid path, a default photo will be chosen for your specified contact.
```

Comment:
should we consider users' POV? they may not necessarily understand what `absolute path` means. Perhaps can refer to the above example and explain? Or simpler, you can guide users how to obtain the absolute path